hook!-full-game/src/Player/States/Wall.gd

```diff
@@ -0,0 +1,39 @@
+extends State
+
+const WALL_SLIDE_SPEED: = 100.0
+const WALL_PULL_STRENGTH: = 300.0
```

Comment:
I should probably remove that, in line 16 I am using it but it doesn't actually do anything because when we jump we switch to the "Move/Air" state, which doesn't apply this horizontal force.